manager/config/manager/manager.yaml

```diff
@@ -35,7 +29,7 @@ spec:
           - name: ENABLE_WEBHOOKS
             value: "true"
           - name: MOVER_IMAGE
-            value: "image-registry.openshift-image-registry.svc:5000/mover/mover:1.0-SNAPSHOT"
+            value: "image-registry.openshift-image-registry.svc:5000/the-mesh-for-data/mover:latest"
```

Comment:
This should by default point to the mover image from ghcr.io. For local testing Aidan has implemented something to override the image repo, we should make it cover this case as well if it doesn't.